drivers/hmis/spec/system/hmis/enrollment_management_spec.rb

```diff
@@ -169,7 +169,8 @@ def make_household(household_id: Hmis::Hud::Base.generate_uuid, enrollment_facto
             user: u1,
           )
           # override garbage factory exit date
-          exit.update!(exit_date: today - 4.days)
```

Comment:
This hits a validation error; I'm not sure why this validation wasn't firing before

Reply:
Weird.
This code points to something that needs to be addressed in the `hmis_hud_exit` factory, to make it not override exit_date if it was passed explicitly. or change line 165 to use hmis_base_hud_exit